ncard_app/views/filters.py

```diff
@@ -140,6 +143,6 @@ def universal_search(self, queryset, name, value):
         for key in student_reverse.keys():
             if value.lower() in key.lower():
                 dict_value = student_reverse[key]
-                qs.add(Q(award_type=dict_value), Q.OR)
+                qs.add(Q(student_type=dict_value), Q.OR)
```

Comment:
Make a similar change here. Don't bother making a reverse dict if you're going to iterate over the whole thing